src/microbe_stage/MicrobeHUD.cs

```diff
@@ -1130,34 +1155,79 @@ private void UpdateATP(float delta)
         atpLabel.HintTooltip = atpText;
     }
 
-    private ICompoundStorage GetPlayerColonyOrPlayerStorage()
+    private ICompoundStorage GetPlayerStorage()
```

Comment:
Is there a reason why this was renamed?

Reply:
For simplicity and so the other method won't be too long `GetPlayerColonyOrPlayerIngestedSizeCount`. But this rename will probably be irrelevant with the comment above.